core/src/test/java/io/seata/core/store/db/DataBaseLockStoreDAOTestByPostgresql.java

```diff
@@ -0,0 +1,301 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.core.store.db;
+
+import io.seata.core.store.LockDO;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.dbcp.BasicDataSource;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author zhangsen
+ * @date 2019/4/26
+ */
+public class DataBaseLockStoreDAOTestByPostgresql {
+
+    static LockStoreDataBaseDAO dataBaseLockStoreDAO  = null;
+
+    static BasicDataSource dataSource = null;
+
+    @BeforeAll
+    public static void start(){
+        dataSource =  new BasicDataSource();
```

Comment:
please do not use the real url to connect database, you can check out other uni test like TableRecordsTest and use MockDriver